WISH/donations.yml

```diff
@@ -0,0 +1,32 @@
+- ticker: WISH
+  recipient: 
+    name: Akaraka Limited
+    address: 77 Robinson Road, #13-00, Robinson 77, Singapore 068896
+    contact: info@akaraka.org.sg
+    ada: 32.26
```

Comment:
The `ada` and `usd` props are indented too much by one level making the CI throw up, can you fix that?